src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -59,9 +60,9 @@ export interface Props {
    */
```

Comment:
Could you address the todo comment on the `items` prop? Especially since it says the array has a specific shape; would be nice to know what it is.